tests/ElementControllerTest.php

```diff
@@ -13,8 +13,6 @@ class ElementControllerTest extends FunctionalTest
 {
     protected static $fixture_file = 'ElementControllerTest.yml';
 
-    protected static $use_draft_site = true;
```

Comment:
Removing because FuntionalTest.use_draft_site is deprecated. This never actually needed to be set to true for the unit test to pass